packages/core/src/types.ts

```diff
@@ -2,20 +2,56 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2019-Present Datadog, Inc.
 
-import type { UnpluginOptions } from 'unplugin';
+// #imports-injection-marker
```

Comment:
⛏️ **small nit**: I don't think this would be easy for a new contributor to understand that the code between these 2 markers gets replaced when the `yarn cli integrity` tool runs so it might be nice to add a comment or something? 
(also I could not find in the contributing.md/readme.md a mention about when to run this tool but maybe it's not meant to?)